website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,127 @@
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1.png)
+<br><br>
+2. The cloud user can directly copy and upload data in the Hadoop’s legacy format, such as Parquet, ORC, RCFile, Avro, and other formats, to OSS, then use DLA to directly point files or folders for which the table is created to query. Then the user may use BI tools for business data analysis and visualization. <br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2.png)
+<br><br>
+3. In order to provide better query performance and low storage cost for subsequent data analysis on OSS, data can be converted into Parquet or ORC format to improve the cost performance of repeated data analysis.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case3.png)
+<br><br>
+
+### Alibaba Use Case Studies: 
+
+#### Customer Case #1: Log Data Analytics
+
+Cloud customers generate log data which is stored on OSS. The challenge is that those log data are unstructured and typically they are in Petabytes of data size. <br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1_logAnalysis.png)
+<br><br>
+So in order to make it available to analyze, DLA’s massive parallel processing engine is the key to business success. It allows users to do full-volume, personalized log data analysis by using DLA’s Hive connector. Additionally log data can be synchronized to OSS with a max delay of 5 minutes with Alibaba Cloud SLS that can feed data to DLA in near real-time.
+<br><br>
+
+#### Customer Case #2: Genetic Data Analytics
+
+
+One of Alicloud’s data vendors is in the Genetic engineering industry. The data volume is gigantic and the data are stored in various data sources. Its raw data containing all bioinformatics for storing gene sequence variations are stored as Variant Call Format(VCF). The challenge is that the Genetic datasets are huge and need to be analyzed promptly. 
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2_gene_data_analytics.png)
+<br><br>
+The solution is to place a huge amount of gene data on OSS for low cost storage service. In the meantime, its reference data that are created as indexing data are stored in the KV system in a distributed NoSQL database by Aliyun Open Table Service(OTS). Now the user may use DLA’s cross data source analytic capability to join the data together to quickly access and analyze for which the resulting query runtime is promising.
+<br><br>
+
+
+# Conclusion
+
+
+Alibaba Data Lake Analytics embraces Presto’s federated query engine capability and has empowered many Alibaba cloud users to experience large scale serverless cloud service. DLA is working to make cloud user experience more transparent for ease of use and ease of management. The next blog will discuss some key technical development works on top of Presto codebase.
```

Comment:
Suggestion:

DLA is working to make the cloud user experience more transparent by improving ease of use and reducing infrastructure management overhead. The next blog will discuss some key technical developments on top of the Presto codebase.

Reply:
fixed